lesson02/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
+  </head>
+  <body>
+    <div>
+      <ul id="js-lists"></ul>
+    </div>
+    <script src="./index.js"></script>
```

Comment:
IMO
Instead of declaring it before the body tag, declare it in the head tag.
I think this one is important to avoid bugs.
[<script> タグに async / defer を付けた場合のタイミング](https://qiita.com/phanect/items/82c85ea4b8f9c373d684)
```suggestion
 <script src="./index.js" defer></script>
```

Reply:
I will definitely add the defer.
Thanks for letting me know!

Reply:
Q
I commented this way
> Instead of declaring it before the body tag, declare it in the head tag.

Is there a reason why you put it before </body>?

Reply:
Sorry...!
Move to the head tag.

Reply:
Thanks for telling me!
[修正内容のコミットIDをリンクでコメントのスレッドに伝える。](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/2#issuecomment-991014358)

Please confirm.→[367fc94](https://github.com/AyakaOnozuka131/JavaScript-lessons/pull/2/commits/367fc94b4150f0e0b5c9feeab96216d8d9be94f9)